schema/spacewalk/common/tables/rhnPackageExtraTagKey.sql

```diff
@@ -0,0 +1,22 @@
+--
+-- Copyright (c) 2019 SUSE LLC
+--
+-- This software is licensed to you under the GNU General Public License,
+-- version 2 (GPLv2). There is NO WARRANTY for this software, express or
+-- implied, including the implied warranties of MERCHANTABILITY or FITNESS
+-- FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv2
+-- along with this software; if not, see
+-- http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+--
+
+CREATE TABLE IF NOT EXISTS rhnPackageExtraTagKey
+(
+    id          NUMBER NOT NULL
+                    CONSTRAINT rhn_pkg_extra_tags_keys_id_pk PRIMARY KEY,
+    name        VARCHAR2(256) NOT NULL,
```

Comment:
I would suggest to add a unique index on name. I think we will search for it a lot.

Reply:
Added index in #1237.